File: Department.py
class Department:
    id: int
    title: str

    def __init__(self, id: int, title: str) -> None: # обвязка(тут инициализатор/конструктор)
        self.id = id
        self.title = title

    def __repr__(self) -> str:
        return f'title: {self.title}  id: {self.id}'

File: department.cs
class Department
{
    public int id;
    public string title;

    public Department(int id, string title) 
    {
        this.id = id;
        this.title = title;
    }

    public override string ToString()
    {
        return $"title: {this.title}  id: {this.id}";
    }
}